astroquery/mpc/core.py

```diff
@@ -1184,7 +1184,11 @@ def _parse_result(self, result, **kwargs):
 
             if len(src) == 0:
                 raise RuntimeError(('No data queried. Are the target '
-                                    'identifiers correct?'))
+                                    'identifiers correct?  Is the MPC '
+                                    'database search working for your '
+                                    'object? The service is hosted at '
+                                    'https://www.minorplanetcenter.net/'
```

Comment:
I actually liked the previous error message more, and at minimum we should use the URL from a variable rather than hard wire in here. Maybe the overall issue should be rather mentioned in the docs? (E.g. as everywhere in astroqeury, if a query doesn't work with the web interface then it's not really an issue for astroquery but either an upstream issue/bug or user error)

Reply:
What if the error instead simply reported what it found?  EmptyResponseError("The query returned no observations.")

Reply:
I think there are two main cases when there are no results. One, that should raise an error is when e.g. the identifiers are incorrect, and the service may even return an error itself. OTOH I can think of valid empty responses when all inputs are correct, just there are simply no observations in the given region/epoch for the given object. 

Here we have the first case, I suppose, and therefore an EmptyResponseError is a good approach (or a ValueError if we also identify the issue with one of the input parameters, but I don't think we are checking on those)